tests/Integration/WorkloadManagementSystem/Test_ParametricJob.py

```diff
@@ -0,0 +1,135 @@
+""" This is a test of the parametric job generation tools
+"""
+
+import unittest
+
+from DIRAC.Core.Base.Script import parseCommandLine
+parseCommandLine()
```

Comment:
pure unittests can't have `parseCommandLine()`: they need to run without configuration. If a configuration is needed, than the test should go in the tests/Integration/WorkloadManagementSystem.